src/main/java/servlets/ResultsServlet.java

```diff
@@ -36,21 +37,20 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet responsible for creating new comments. */
+/** Servlet responsible for returning search results. */
 @WebServlet("/results")
 public class ResultsServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String userQuery = request.getParameter("user-query");
     Query query = new Query("Recipe").addSort("timestamp", SortDirection.DESCENDING);
-
     query.setFilter(generateFiltersFromQuery(formatQueryAsList(userQuery)));
 
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     PreparedQuery results = datastore.prepare(query);
 
-    List<TestRecipe> testRecipes = new ArrayList<>();
+    List<TestRecipe> testRecipes = new LinkedList<>();
```

Comment:
What is the rationale behind changing to LinkedList?

Reply:
Better insertion time, doesn't have the ArrayList space resizing issue. I don't really need an ArrayList here; any list will do, so I switched over.